package.json

```
@@ -37,7 +37,7 @@
   "scripts": {
     "build": "THEME=\"light\" node esbuild.config.js",
     "build:css": "bin/link; THEME=\"light\" yarn build; yarn light:build:css; yarn light:build:mailer:css",
```

```suggestion
    "build:css": "THEME=\"light\" yarn build; yarn light:build:css; yarn light:build:mailer:css",
```

I'd like to see what happens in CI if we remove this call to `bin/link`. I think it's clobbering the branch-matched core gems that we link earlier in the CI process. (Also, I don't understand why we'd ever want that call to `bin/link` there in the first place. It seems like an internal-BT-dev tool thing that we shouldn't need as part of normal operation.)

---

`bin/link` is part of normal operation for Bullet Train apps, it's how CSS/JS assets get resolved from engines/gems. Since the gems get symlinked into `tmp/gems`, the Tailwind config can then pick them up: https://github.com/bullet-train-co/bullet_train-core/blob/dab52ad22c932a4c7003c8747ba1c668c39f2444/bullet_train-themes-light/tailwind.light.config.js#L11-L14

I'm not sure what the alternatives for engine assets would be, if we want to avoid a too large prebuilt Tailwind compiled CSS file.